remoteworkitem/github.go

```diff
@@ -13,6 +15,11 @@ type Github struct {
 	Query string
 }
 
+// Github doesn't allow commiting actual tokens no matter how
+// less privleges the token has.
+var camouflagedAccessToken = "751e16a8b39c0985066-AccessToken-4871777f2c13b32be8550"
```

Comment:
I meant extract them so they can be set from the command line/env like the others. :)